src/components/ProductCarousel/index.tsx

```diff
@@ -26,10 +27,18 @@ const ProductCarousel: React.FC<ProductProps> = ({ product }) => (
         <p className='text-sm font-semibold md:text-lg md:font-medium'>${product.price}</p>
         <span
           className={`rounded px-2 py-1 text-xs text-white ${
-            product.isRestored ? 'bg-green-500' : 'bg-blue-500'
+            product.condition === 'A'
```

Comment:
You should keep the different product conditions in a constant and use it in the filters (`conditionMapping` const) and everywhere else you try to see what the condition is